app/src/main/java/cm/aptoide/pt/analytics/analytics/AnalyticsManager.java

```diff
@@ -46,53 +60,139 @@ public void logEvent(Map<String, Object> data, String eventName, Action action,
     }
   }
 
+  /**
+   * <p> Log a {@code url} to the {@link HttpKnockEventLogger}.</p>
+   *
+   * @param url The url to log.
+   */
   public void logEvent(@NonNull String url) {
     knockEventLogger.log(url);
   }
 
+  /**
+   * <p>Setup the {@code EventLogger(s)} that need to be setup, like AptoideBiEventLogger</p>
+   */
   public void setup() {
     for (Map.Entry<EventLogger, Collection<String>> loggerEntry : eventLoggers.entrySet()) {
       loggerEntry.getKey()
           .setup();
     }
   }
 
+  /**
+   * <p> Starts the Flurry session allowing to log events to Flurry.</p>
+   */
```

Comment:
Phrasing. Would go with something like "Start the session allowing to log Flurry events"